security/authorization/go.mod

```diff
@@ -17,5 +17,6 @@ require (
 	github.com/envoyproxy/protoc-gen-validate v0.1.0 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/stoewer/go-strcase v1.2.0 // indirect
-	golang.org/x/text v0.3.7 // indirect
+	golang.org/x/net v0.4.0 // indirect
```

Comment:
are we adding `golang.org/x/net` as a dependency for the new v0.5.0 for `golang.org/x/text`?

Reply:
Interesting.  TBH I just blindly did `go get ..x/net` and assumed `go mod tidy` would remove it if it wasn't needed by anything in practice.  These are indirect dependencies, meaning something we depend on apparently depends on it.  It seems like a good idea to bump it, too, in case they didn't, and in case it matters.

Reply:
SGTM! 🚢

Reply:
The `golang.org/x/` projects started to tag "SemVer®" releases, erm.. by not really following SemVer, but just tagging new releases periodically (CalVer disguising as SemVer?); well, mostly because they have circular dependencies; see https://github.com/golang/go/issues/48523